src/libAtomVM/port.c

```diff
@@ -0,0 +1,98 @@
+/***************************************************************************
+ *   Copyright 2018 by Davide Bettio <davide@uninstall.it>                 *
```

Comment:
You are the author here, you can change this :)

Reply:
fixed